[BOP-1236] Remove instructions on direct config resource editing (#125)
Mirantis/mke-docs: content/docs/operations/telemetry.md

## Enable telemetry through the MKE CLI

> In the MKE config file, set the `telemetry.enabled` field to `true` to enable telemetry.

Reviewer: ```suggestion
1. Access the MKE configuration file.

2. Set the `telemetry.enabled` field to `true`.
```

Author: done

> ```yaml
> spec:
>   tracking:
>     enabled: true

Reviewer: ```suggestion
   ```yaml
   spec:
     tracking:
       enabled: true
   ```

Reviewer: The point here being to have the codeblock align under the step text and not the step number.

Author: done

```

> Apply the new settings with `mkectl apply`.

Reviewer: ```suggestion
3. Run the  `mkectl apply` command to apply the new settings.
```

Author: done

It will take a few moments to reconcile the change in the
configuration, after which MKE will thereafter transmit key usage data to

> Mirantis by way of a secure Segment endpoint.

Reviewer: Please edit the sentence to read:

"After a few moments, the change will reconcile in the configuration. From this point onward, MKE will transmit key usage data to Mirantis by way of a secure Segment endpoint."

Author: done
